tools/utils/utils.js

```diff
@@ -44,11 +44,15 @@ exports.parseUrl = function (str, defaults) {
   // for consistency remove colon at the end of protocol
   parsed.protocol = parsed.protocol.replace(/\:$/, '');
 
-  return {
+  var ret = {
```

Comment:
We should use `let` and not `var`

Reply:
@filipenevola I'm easy with either - but `var` is used everywhere else in this file. I kept it the same for consistency.